music_this_week_app/backend/spotifyHandler.py

```diff
@@ -49,7 +68,12 @@ def filter_artist(self, artist_name):
         # Check if the artist is in the database
         try:
             artist_entry = Artist.objects.get(name = artist_name)
-            artist_uri = artist_entry.spotify_uri
+            self.artists_uris_found += 1
+            if not artist_entry.top_tracks == None:
+                top_tracks = artist_entry.top_tracks.split(',')
+                self.cached_tracks += top_tracks
+            #raise a value error to let caller know artist found in DB
+            raise ValueError()
```

Comment:
2 ways I thought of doing this:

1. have filter_artist return a tuple, but that would mean a similar design I implemented before that you felt was confusing
2. factor out artist lookup and do it directly from filter_artistS  . impelement the logic in filter_artists